scripts/process_json.py

```
@@ -55,17 +62,31 @@ def parse_text(text) -> None:
 
     return text, source_type, version, date
 
+
 def load_card_names(*card_files):
+    """
+    Load card codes and their corresponding names from one or more JSON card data files.
+    
+    Parameters:
+    	card_files: Paths to JSON files containing card data.
```

Ruff (0.11.9)

Indentation contains mixed spaces and tabs

(E101)